vbfnlo.spec

```diff
@@ -0,0 +1,24 @@
+### RPM external vbfnlo 3.0.0beta5
+Source: http://www.itp.kit.edu/~vbfnloweb/archive/vbfnlo-%{realversion}.tgz
```

Comment:
@Andrej-CMS , why not use https://github.com/vbfnlo/vbfnlo  to download it? You can use
```
git+https://github.com/vbfnlo/%{n}.git?obj=master/v%{realversion}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
```

and then can do
```
%setup -q -n %{n}-%{realversion}
```

Reply:
Dear @smuzaffar ,
I get now an error that states that the Manual.pdf file cannot be compiled from TeX file. Do we have in cmsdist some latex package that need to build first?
[vbfnlobuild.txt](https://github.com/cms-sw/cmsdist/files/2940692/vbfnlobuild.txt)

Reply:
@smuzaffar 
do you have any idea in regards of the question above?

Reply:
@Andrej-CMS , I do not see build errors if I run `autoreconf -vi` before configure